### ISSUE TITLE
Silverpine Forest Quest Chain

### ISSUE THREAD

**Issue by Lorddraconis** [Atlantiss/BugTracker#9726]
**Description**: Hello! So I'm sure one of the devs has at least heard about the Godfrey trio in Silverpine forest? Well the quest chain in Silverpine forest is completely undoable.

**Current behaviour**: Well the problem is Godfrey I think, or one of the two NPCs following you with him, is suppoused to give a quest were you go capture Lorna Crowly. Instead he doesn't give the quest and is stacked on top of the other two NPCs. Making giving in their quest very hard. The entire area seems buggy as I heard the Pyrewood village is suppoused to be phased meaning it is half like a instance. 

**Expected behaviour**: Godfrey&Co are suppoused to be spread out and not on top of each other. The are suppoused to give out the quest on her majesty's secret service quest. Which is one of the finale quest in Silverpine forest.

**Steps to reproduce the problem**:

1. Make a new undead character.
2. Level up and go Questing in Silverpine Forest
3. Do the Silverpine forest quest chain.

**Revision**: Not exactly sure what this means, I am a bit of a Noob IRL. I assume it is the current patch of the server? If so then it is 4.3.4

**Comment by tomatosz** [comment]
Ok, so if I understand correctly, the problems are:
1. There is no phasing
2. the npc that follow you are stacked on top of each other
3. [On Her Majesty's Secret Service](http://www.wowhead.com/quest=27594/on-her-majestys-secret-service) is not obtainable?
The first two have been reported already (#4249).

Also, the revision is more specifically the version the server is currently at in terms of fixes etc. That's rev. 5550 right now (you can see that by typing .server info in game, or by checking https://atlantiss.eu/changelog)

**Comment by Lorddraconis** [comment]
Yes. And there after On Her Majesty's Secret Service there is another quest
that should be the end of the zone. But as Godfrey doesn't give the quest
(I honestly think that would be a simple fix right? Just to somehow fix the
problem of him not giving the quest.) The first two problems aren't that
big a problem as the quest are still doable but the last basically stops
you from doing the last two quest.

**Comment by tomatosz** [comment]
Are you sure you had [Sowing Discord](http://www.wowhead.com/quest=27580/sowing-discord) and [7th Legion Battle Plans](http://www.wowhead.com/quest=27577/7th-legion-battle-plans) finished? I can't reproduce the problem.

**Comment by Lorddraconis** [comment]
Yes. I am sure I did both of those quest. My friend is also having the
exact same problem. We did Sowing Discord and 7thLegion Battle Plans but
Godfrey never offered us the next Quest.
